### PR TITLE
fix(DB/SAI): NPC Nancy Vishas

### PR DESCRIPTION
**Pull request by Stifler82**
## Changes Proposed:
- Fixed Waypoints NPC: [Nancy Vishas](https://wowgaming.altervista.org/aowow/?npc=3984)
- Fixed SAI NPC: [Nancy Vishas](https://wowgaming.altervista.org/aowow/?npc=3984)
- Fixed SAI NPC: [Grandpa Vishas](https://wowgaming.altervista.org/aowow/?npc=3985)

## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/222
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4942

## Tests Performed:

- Tested in game and working
- Tested on Windows 10 (64)
- **Video Test:** [Nancy Vishas](https://youtu.be/5WGoMcWPBao)

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Kitzunu — APPROVED** [review]
sql looks fine

File: data/sql/updates/pending_db_world/rev_1617265345631987700.sql [hunk 1]
```diff
@@ -12,7 +12,7 @@ INSERT INTO `waypoints` VALUES
 
 DELETE FROM `smart_scripts` WHERE (`entryorguid` = 398400) AND (`source_type` = 9)
 
-UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` IN (3984, 3985));
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (entry IN (3984, 3985));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1617265345631987700.sql:15]: why?

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql:15]: because it gave me an error, and i tried like this..

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql:15]: in my DB it work in both ways

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql:15]: idk why it gave this error

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql:15]: That's weeeeeird, ok ^^

File: data/sql/updates/pending_db_world/rev_1617265345631987700.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+DELETE FROM `waypoints` WHERE (`entry` IN (3984));
+INSERT INTO `waypoints` VALUES
+(3984, 1, 888.1400, -121.8399, 38.3880, 'Nancy Vishas'),
+(3984, 2, 891.5708, -114.1264, 38.2004, 'Nancy Vishas'),
+(3984, 3, 909.1765, -90.91162, 33.7845, 'Nancy Vishas'),
+(3984, 4, 891.5708, -114.1264, 38.2004, 'Nancy Vishas'),
+(3984, 5, 888.1400, -121.8399, 38.3880, 'Nancy Vishas');
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 398400) AND (`source_type` = 9)
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1617265345631987700.sql:13]: Btw it is failing here due to missing colon `;`

File: data/sql/updates/pending_db_world/rev_1617265345631987700.sql (hunk 1 above)
Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql]: i putted it like before.. give a look

File: data/sql/updates/pending_db_world/rev_1617265345631987700.sql (hunk 2 above)
Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql:13]: looooooooooooool i didn't noticed it... ty @Kitzunu

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql]: it gave me this error before:
```ERROR 1064 (42000) at line 28586: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` IN (3984, 398' at line 3```

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617265345631987700.sql]: because this error i made that change..

**Comment by pklloveyou** [comment]
Tested

**Pull request merged**